README.md

```diff
@@ -56,3 +56,9 @@ function App() {
 
 export default App;
 ```
+
+To use Answers React Components stylesheet as an alternative to Tailwind CSS, import /bundle.css as follows:
```

Comment:
It's not really an alternative to Tailwind CSS. This stylesheet was built with Tailwind. We make this available so others aren't forced to use Tailwind to get out styling. 

I'd update the copy to something like: "To use the Component Library's Styling without adding Tailwind to your project, add the following import: "